SpatialGDK/Source/SpatialGDK/Public/Utils/LayerInfo.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include "LayerInfo.generated.h"
+
+USTRUCT()
+struct FLayerInfo
+{
+	GENERATED_BODY()
+
+		UPROPERTY()
+		FName Name;
+
+	// Using TSoftClassPtr here to prevent eagerly loading all classes.
+	/** The Actor classes contained within this group. Children of these classes will also be included. */
+	UPROPERTY(EditAnywhere, Category = "SpatialGDK")
+		TSet<TSoftClassPtr<AActor>> ActorClasses;
+
+	FLayerInfo() : Name(NAME_None)
```

Comment:
Nit: I prefer constructors at the start of the class. Don't have to move unless you agree.

Reply:
Moved as per previous comment.